packages/website/src/search/componentTitleList.csv

```diff
@@ -0,0 +1,98 @@
+Iconography
```

Comment:
The list is outdated, some stuff was removed in during cleanup

Reply:
yeah I know :see_no_evil:  
I don't think I can remove candidates from the model but I can create another one :+1: